feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsScreen.kt

```diff
@@ -78,6 +80,18 @@ fun SettingsList(uiState: SettingsUiState, viewModel: SettingsViewModel) {
         setCaptureMode = viewModel::setCaptureMode
     )
 
+    // todo: b/313647247 - query device and disable setting if stabilization isn't supported.
+    PreviewStabilizeSetting(
```

Comment:
Can we also add a UI affordance (a small icon) to `PreviewScreen` to show when stabilization is enabled? We already have the `video_stable` material icon we can use. This will also allow you to write some tests to check that changing the settings in the SettingsScreen has some effect in the PreviewScreen.

Reply:
yes but it will need to have information from CameraXCameraUseCase to know if stabilization is actually supported.

Right now the UI on preview/quicksettings reflect the information in the CameraAppSettings (currentCameraSettings). I think for now we can wait until the UDF refactor is complete, and add the supported information to the config

Reply:
As discussed offline, we should check once when starting up which stabilization modes are available and only present those as options in the quick settings / settings. Once those settings are selected in the UI, we can assume they're being applied and show an icon, since we've already checked that they're supported.

This may be sufficient, but later on we should also handle cases where we set a setting and CameraXCameraUseCase tells us that the camera wasn't able to apply the setting. This will likely be handled by showing some kind of error in the UI.